src/components/ReactionGameResultDisplay.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+    <div class="message-box">
+        {{ message }}
+        <div v-if="reactionTime !== null" class="ShowreactionTime">
+            Your reaction time was {{ reactionTimeInSeconds }} seconds.
+        </div>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'ReactionGameResultDisplay',
+    props: {
+        message: String,
+        reactionTime: Number,
+    },
+    computed: {
+        reactionTimeInSeconds() {
+            return this.reactionTime !== null ? (this.reactionTime / 1000).toFixed(3) : "0.000";
+        },
+    },
+};
+</script>
+
+<style scoped>
+.message-box {
+    margin-top: 20px;
+    background-color: #d9edf7;
+    border-radius: 10px;
+    font-size: 25px;
+    text-align: left;
+    color: #333;
+    width: 430px;
+    height: 120px;
+    padding: 20px;
+    gap: 10px;
+}
+
+.ShowreactionTime {
+    padding-top: 20px;
+}
+
+@media (max-width:768px) {
+    .message-box {
+        margin-top: 20px;
```

Comment:
there are multiple styles that change according to the screen width, any reason for writing down again-again? @Shashank-002